maintainers/maintainer-list.nix

```diff
@@ -11479,4 +11479,10 @@
     github = "zupo";
     githubId = 311580;
   };
+  nfitzgibbon = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add nfitzgibbon`` and sort it alphabetically.

Reply:
This was in a separate commit within this PR -- I've now also changed the handle to match my github username and alphabetized it in the latest commit.